workflow/scripts/krakenuniq_abundance_matrix.R

```diff
@@ -13,9 +13,9 @@ for(i in 1:length(krakenuniq_outputs))
  names(df[[i]])[1]<-"Pers_Reads"
  df[[i]]$SAMPLE<-krakenuniq_outputs[i]
  df[[i]]<-na.omit(df[[i]])
- df[[i]]<-df[[i]][df[[i]]$kmers>1,]
+ df[[i]]<-df[[i]][df[[i]]$kmers>args[3],]
```

Comment:
Same as above, and the filtering can be skipped altogether if this is run on `krakenuniq.output.filtered` instead